csp_billing_adapter/local_csp.py

```diff
@@ -83,3 +83,9 @@ def get_account_info(config: Config) -> str:
         'arch': 'x86_64',
         'cloud_provider': 'local'
     }
+
+
+@csp_billing_adapter.hookimpl
+def get_version():
+    version = csp_billing_adapter.__version__
+    return ('local_csp_plugin', version)
```

Comment:
Just a thought, but perhaps this `local_csp_plugin` should be stored as a "constant" value that can be imported into the test script below, rather than using the text string in both places?

Just a thought since the "local_csp" plugin is a bit of a misnomer, and is really the "test_csp" or "unittest_csp", and could easily be confused with the `local` plugin... In case we ever wanted to rename it.

Reply:
Agree on name change.

Reply:
I don't see much value in making the string a constant. If it ends up getting used multiple places in code then a var makes sense.